src/smart-components/workflow/workflows.js

```diff
@@ -53,31 +56,55 @@ const prepareChips = (filterValue, intl) => filterValue ? [{
 
 const initialState = (filterValue = '') => ({
   filterValue,
-  isOpen: false,
   isFetching: true,
-  isFiltering: false
+  isFiltering: false,
+  selectedWorkflows: [],
+  selectedAll: false,
+  rows: []
 });
 
 const workflowsListState = (state, action) => {
   switch (action.type) {
+    case 'setRows':
+      return {
+        ...state,
+        rows: action.payload,
+        selectedAll: false
+      };
     case 'setFetching':
-      return { ...state, isFetching: action.payload };
+      return {
+        ...state,
+        isFetching: action.payload,
+        selectedWorkflows: [],
```

Comment:
why do we have to reset the selection? I think this prevents bulk actions across multiple pages now. We will lose the object reference.

Reply:
And do we want to allow to select WFs across multiple pages? I can imagine it would be hard for users to clear the selection, they wouldn't see what they delete and I can't find using this pattern in any of my email clients. :D

Reply:
Yes, we do. It was like that for a reason. There should be even a bulk selection that would select all on the current page so you can quickly go through and select everything (not this task). There are also talks to have "select totally all" but that is due to pagination not really possible atm.

Reply:
Ok, I will fix this functionality in this PR.

Reply:
:+1: let me know once it's done. I have no other comments about this PR.

Reply:
Fixed. :eagle: